sources/assembly-data/ATTR_assembly.types.yaml

```diff
@@ -238,6 +327,8 @@ attributes:
     type: long
     units: bases
   chromosome_count:
+    description: Chromosome count
+    long_description: Total number of chromosome types identified in an assembly submission. The chromosome count is equal as the haploid chromosome number of a taxon when the sequenced individual belongs to the homogametic sex. When the sex of the sequenced sample corresponds to the heterogametic sex of a taxon, then the chromosome count ican be lower, higher or equal the haploid number, depending on the karyotype of the individual 
```

Comment:
use of "chromosome types" is confusing here
omit "submission"
is equal _to_ the haploid...
ican -> can
higher or equal _to_ the haploid...